consumer.go

```diff
@@ -834,6 +855,7 @@ func (bc *brokerConsumer) handleResponses() {
 			// not an error, but does need redispatching
 			Logger.Printf("consumer/broker/%d abandoned subscription to %s/%d because %s\n",
 				bc.broker.ID(), child.topic, child.partition, result)
+			child.replicaInited = false
```

Comment:
I wonder if this needs a locking mechanism? seems unsafe
are we sure there are no multiple go routines accessing this?

Reply:
Checked.  it it's fine to be accessed like this.